Tests/ML/histopathology/models/test_transforms.py

```diff
@@ -126,8 +126,10 @@ def test_cached_loading(tmp_path: Path) -> None:
 
 @pytest.mark.skipif(not os.path.isdir(TCGA_CRCK_DATASET_DIR),
                     reason="TCGA-CRCk tiles dataset is unavailable")
-@pytest.mark.parametrize('use_gpu', [False, True])
-def test_encode_tiles(tmp_path: Path, use_gpu: bool) -> None:
+@pytest.mark.parametrize('use_gpu , chunk_size',
+                         [(False, 0), (False, 2), (True, 0), (True, 2)]
+                         )
+def test_encode_tiles(tmp_path: Path, use_gpu: bool, chunk_size: int) -> None:
```

Comment:
This test relies on datasets being mounted, and this is only the case on some of people's individual VMs. This makes the test not particularly useful. We should talk about a better way of creating datasets for testing - mocking being one obvious option.

Reply:
Agreed with you - most of the tests for histo relies on this strategy